### DISCUSSION TITLE
Ring API rate limit

### DISCUSSION THREAD

**Discussion by flaced** [dgreif/ring#1047] [Q&A] [answered]
Hi folks, just wanted to quickly ask if some of you know about a rate limit in this API 

Thanks,
Marcel

**Comment by dgreif** [comment, marked as answer]
I do not know of any specific rate limits, though I suspect they do exist. The homebridge-ring plugin does polling for device status (sometimes once per second) and has never hit any issues, but I think if you send thousands of requests per second you will likely get limited

**Reply by flaced** [reply]
All right, thanks for the answer!
